projects/components/src/textarea/textarea.component.ts

```diff
@@ -29,6 +31,12 @@ export class TextareaComponent implements OnInit {
   @Input()
   public disabled: boolean | undefined;
 
+  @Input()
+  public rows: number = 2;
+
+  @Input()
+  public minHeight?: number;
```

Comment:
Rather than having an explicit input, can we have the text area stick to the size of its parents so you can style the `<ht-textarea>` the same way you would style it directly? In other words, assigning a minHeight to the component is the desired way of accomplishing this rather than pushing styles through inputs.

Reply:
Done. Let me know your thoughts about my approach.